nncf/api/composite_compression.py

```diff
@@ -224,6 +225,15 @@ def prepare_for_export(self) -> None:
             stripped_model = ctrl.strip_model(stripped_model)
         self._model = stripped_model
 
+    @property
+    def loss(self) -> CompressionLoss:
+        return self._loss
```

Comment:
Could you check that using 0 value does not lead to "zero_loss" issue here https://github.com/openvinotoolkit/nncf/blob/29a596c98cab03a08591e5e1006430f671d50f1c/nncf/api/composite_compression.py#L57